packages/engine/Source/Shaders/GlobeVS.glsl

```diff
@@ -114,7 +114,8 @@ vec4 getPositionMorphingMode(vec3 position, float height, vec2 textureCoordinate
     float yPositionFraction = get2DYPositionFraction(textureCoordinates);
     vec4 position2DWC = vec4(height, mix(u_tileRectangle.st, u_tileRectangle.pq, vec2(textureCoordinates.x, yPositionFraction)), 1.0);
     vec4 morphPosition = czm_columbusViewMorph(position2DWC, vec4(position3DWC, 1.0), czm_morphTime);
-    return czm_modelViewProjection * morphPosition;
+    vec4 positionEC = czm_modelView * position;
```

Comment:
This is significantly changing the output. For a consistent result, this could be:
```glsl
vec4 positionEC = czm_modelView * morphPosition;
```

However, we may want to avoid using the `positionEC` name for modified positions, because that name has a very specific meaning in most of the rest of the code.